### PR TITLE
added more routes and fixed funcs nanmes

### PR DESCRIPTION
**Pull request by markfridman**

### CODE REVIEW

File: src/services/buckets.service.js [hunk 1]
```diff
@@ -7,11 +7,19 @@ class BucketService {
     return Bucket.find({})
   }
   createBucket(query) {
-    console.log('query', query)
     return Bucket.create({ 
       ...query
```
Comment (NitayRabi) [inline src/services/buckets.service.js:11]: You don't need to deconstruct the query (`{...query}`) - just pass it to create

File: src/routers/buckets.js [hunk 2]
```diff
@@ -2,33 +2,49 @@ import express from 'express';
...
+        console.log('There was a problem with your query: ', e)
+        return next({ status: 500, data: e });
+      }
+    });
+
+    // ----------------  CREATE  ----------------//
+    this.express.route('/create').post(async (req, res, next) => {
+      try {
+        // const data = await this.services.rounds.getBuckets(req.query);
+        const data = await this.services.buckets.createBucket(req.body);
```
Comment (NitayRabi) [inline src/routers/buckets.js:42]: Moongoose handles validations? will i get errors if I'm missing required data for e.g?

File: src/services/rounds.service.js [hunk 3]
```diff
@@ -2,14 +2,31 @@ import mongoose from 'mongoose';
...
+  
   createRound(query) {
-    return Round.create({ 
+    return Round.create({
       ...query
     });
   }
+
+  byEventId(event_id, based_on_event_id) {
+    return Round.find({
```
Comment (NitayRabi) [inline src/services/rounds.service.js:16]: can you throw error in case you don't get `event_id` and `based_on_event_id`?

**Review by NitayRabi — CHANGES_REQUESTED** [review]

**Comment by NitayRabi** [comment]
Please add group_type (Production, camp, art installation) to round model using constants

**Review by NitayRabi — APPROVED** [review]

File: src/routers/buckets.js (hunk 2 above)
Comment (markfridman) [reply src/routers/buckets.js:42]: yes, it depends on your model, if boolean "requiered" is mentioned as true for each key

File: src/services/rounds.service.js (hunk 3 above)
Comment (markfridman) [reply src/services/rounds.service.js:16]: yes

**Pull request merged**